Modules/_testinternalcapi.c

```diff
@@ -35,6 +36,38 @@ get_recursion_depth(PyObject *self, PyObject *Py_UNUSED(args))
 }
 
 
+static PyObject*
+codecs_unregister(PyObject *self, PyObject *args)
```

Comment:
For me, it would not be unreasonable to expose this function as _codecs.unregister().

I would prefer to define this function in Python/codecs.c.

Reply:
Hm, some outside users will call this api? I am not sure about it.

Reply:
You can now remove this function, since you wrote codecs.register() :-)

Reply:
Copy that, updated.